mantidproject/mslice
# Split WorkspaceManager into tabs

**Not to be merged until after #208**

Implements new `WorkspaceManager` design with tabs for each workspace type. The save / rename / combine controls remain but have moved. The `Load` command is now handled by the `DataLoaderWidget`, not the `WorkspaceManager`. 

**To test:**

- Using the `Data Loading` tab, load workspaces of each type (`Workspace2D`, `MDEventWorkspace`, `MDHistoWorkspace`) and check they are placed in the correct `WorkspaceManager` tab.
- Also check that the save / rename / combine controls function as previously.

Fixes #207
